python-connectors/pi-system_event-frames-search/connector.py

```diff
@@ -5,6 +5,7 @@
 from osisoft_constants import OSIsoftConstants
 from safe_logger import SafeLogger
 from osisoft_plugin_common import PISystemConnectorError, RecordsLimit, get_credentials, build_requests_params, assert_time_format, get_advanced_parameters
+# from osisoft_pagination import Pagination
```

Comment:
should this be removed or uncommented ?

Reply:
Indeed I'm looking at changing how the pagination is done throughout the whole plugin but it's not completed yet. I'll remove it for now...

Reply:
Removed in [0e8d31f](https://github.com/dataiku/dss-plugin-osisoft/pull/17/commits/0e8d31feacfaa01e6ac699ff34e6a6d07754c94f)